### ISSUE TITLE
editorial communications communicated

### ISSUE THREAD

**Issue by bruce-usab** [w3c/maturity-model#214]
Whilst [Equity CG](https://www.w3.org/community/equity/) is working through AMM to suggest verbiage for incorporation of equity and sustainability, I noticed the following line under [3.1 Communications](https://w3c.github.io/maturity-model/#Communications) which read awkwardly to me:

> Also, accessibility information about the communications may be communicated so that users are aware of the extent to which they can interact with content.

I suggest instead something like:

> Information about the accessibility of communications should be proactively conveyed so that users are aware of the extent to which they can interact with content.

**Comment by Helixopp** [comment]
We agree with your issue and suggestion but would like to add a reference from How To Make Content Usable Guide related to consistent placement.
